packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -254,6 +254,8 @@ export const LoadImage = (
     };
 
     const cspHandler = (err: any) => {
+        if (err.blockedURI !== img.src) { return; }

```

Comment:
Just wondering if it is always the same here or if it could be converted like absolute / relative ???

Reply:
`blockedURI` should be absolute, and I believe `img.src` is absolute also, at least in the tests I did on some modern browsers. I don't recall if this has been something that has been changed over the years though.

Reply:
I can update it to:
```js
if (err.blockedURI !== new URL(img.src, location.href).toString()) { return; }
```
if you think that would be more robust.

Reply:
Actually, here's an edge case, which doesn't really make sense to do with a URL fragment character, but could affect other characters if they are escaped, particularly unicode characters:

Adding a fragment to the image URL, e.g. `#test`, will show `#test` in the image's `src` property, but the `blockedURI` will show `%23test`, at least in Chromium-based browsers.

Reply:
They should both be absolute

Reply:
@sebavan - can I resolve this? or is there any other concern here?